### PR TITLE
additional bars and anchor text formatting

### PR DESCRIPTION
**Pull request by r1b**
* Now there are outbound links to both map and bar homepage w / descriptive anchor text
* Added two places i like

### CODE REVIEW

File: source/_posts/bars-to-code-at.md [hunk 1]
```diff
@@ -75,8 +75,8 @@ making Sugarburg a good place to work or meet regularly. The bar is large and
 Bourbon-oriented, so Sugarburg is also a great place to hang out late into the
 night.
 
-### Boobie Trap
-Bushwick [Link](https://goo.gl/maps/Q3CUv7sTRVE2)
+### [Boobie Trap](http://www.boobietrapbrooklyn.com/)
+[Bushwick](https://goo.gl/maps/Q3CUv7sTRVE2)
 
 [Robert Jensen](http://you.could.use.some.r1b.solutions/) says that Boobie Trap
```
Comment (ptsteadman) [inline source/_posts/bars-to-code-at.md]: You should make it consistent, either r1b or Robert Jensen

File: source/_posts/bars-to-code-at.md [hunk 2]
```diff
@@ -121,6 +121,24 @@ former Computer Lab Writer in Residence [Theo
 Thimo](https://twitter.com/theoooeooo), who often uses it as a place to read and
 write. I enjoyed the simple veggie burger and wine selection.
 
+### [Rebecca's](https://www.facebook.com/rebeccasbushwick/)
+[Bushwick](https://goo.gl/maps/QtoNcJh9ky52)
+
+Rebecca's is r1b's favorite bar in Bushwick. During the day it is a chill place
```
Comment (ptsteadman) [inline source/_posts/bars-to-code-at.md]: You should make it consistent, either r1b or Robert Jensen

File: source/_posts/bars-to-code-at.md [hunk 3]
```diff
@@ -121,6 +121,24 @@ former Computer Lab Writer in Residence [Theo
...
+at his phone and imbibing $5 beer / shot combos at the end of a long day. Wifi
+is available and there is plenty of seating at the bar, tables and couches.
+
+### [Brooklyn Kava](http://www.brooklynkava.com/)
+[Bushwick](https://goo.gl/maps/g3qLjPmbcT72)
+
+Brooklyn Kava is not your typical bar - they serve psychoactive root beverages
+in lieu of booze. If you haven't had the chance to try kava or kratom I highly
+recommend Brooklyn Kava for your first experience. Regardless, the space is an
+excellent place to get work done. It is typically quiet & calm with lots of
```
Comment (ptsteadman) [inline source/_posts/bars-to-code-at.md]: & -> and (to keep writing styles consistent)

File: source/_posts/bars-to-code-at.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Seven Bars to Code at in Brooklyn
+title: Nine Bars to Code at in Brooklyn
```
Comment (ptsteadman) [inline source/_posts/bars-to-code-at.md]: add "Great" to make it more likely the bars will include it in their social

File: source/_posts/bars-to-code-at.md (hunk 2 above)
Comment (r1b) [inline source/_posts/bars-to-code-at.md]: I'll use r1b

**Pull request merged**